browser/js/common/factories/ReviewFactory.js

```diff
@@ -0,0 +1,17 @@
+app.factory('ReviewFactory', function ($http) {
+  return {
+    createReview: function (stars, review) {
+      return $http.post('/api/v1/reviews', {rating:stars, description:review})
+      .then(function (res) {
+        return res.data;
+      });
+    },
+
+    getReviews: function(songId){
+      return $http.get('/api/v1/reviews')
+      .then(function(res){
+        return res.data;
+      })
+    }
```

Comment:
Instead of a getAllReviews method, should there be a getReviewsByUser, which gets all the review for the user, and a getReviewsBySong, which gets all the reviews for the song (this would probably be used in the single-product page view)